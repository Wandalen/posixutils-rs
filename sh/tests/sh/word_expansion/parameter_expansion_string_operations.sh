VAR=ababcbc NULL=''

echo ${#VAR}
echo ${#NULL}
echo ${#UNSET}
echo ${VAR%}
echo ${VAR%b*}
echo ${VAR%%}
echo ${VAR%%b*}
echo ${VAR#}
echo ${VAR#*b}
echo ${VAR##}
echo ${VAR##*b}