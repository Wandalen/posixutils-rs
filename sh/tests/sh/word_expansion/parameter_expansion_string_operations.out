7
0
0
ababcbc
ababc
ababcbc
a
ababcbc
abcbc
ababcbc
c
